selectors-4/deferred-for-level-5

```diff
@@ -127,3 +127,14 @@ The local link pseudo-class '':local-link''</h3>
 		What about if the document url has less than N segments,
 		and the link is same-page?
 		Should "null segments" count as matching, or not?
+
+<h3 id="custom-pseudo">Exposing custom state: the '':state()'' pseudo-class</h3>
+
+  The <dfn selector>:state()</dfn> pseudo-class takes an argument and matches
+  custom elements whose exposed custom states include the argument.
+
+  The grammar of the '':state()'' pseudo-class is:
+
+  <pre class=prod>:state( <ident> )</pre>
+
+  TODO add some text explaining that this is defined in the HTML spec? The :user-valid pseudo class is like this too: https://github.com/w3c/csswg-drafts/pull/9308
```

Comment:
I think this can be:

```suggestion
 The exact matching behavior of '':state()'' pseudo-class is defined by the host language. See <a spec=html lt="custom" selector>HTML's definition</a>.
```